src/lib/items/Item.js

```diff
@@ -414,7 +414,6 @@ export default class Item extends Component {
 
   componentDidUpdate(prevProps) {

```

Comment:
Out of curiosity and because I'm not yet very familiar with the internal codebase of `react-calendar-timeline`. Should we not call `componentWillUnmount` to unsubscribe from any `interact` event listeners on the `item`?

Reply:
I dont think interactjs has an unmount or cleanup call

Reply:
Looking at the [docs](https://interactjs.io/docs/faq/#remove-%2F-destroy-%2F-release), it looks like there is a method: `interact(target).unset();`

Reply:
interesting. I will open an issue to look into that

Reply:
I actually can't find the documentation for `unset` could you please link it here?